packages/mds-utils/tests/state-transition-expected.ts

```diff
@@ -0,0 +1,198 @@
+export const expectedTransitions : { [key: string] : boolean } = {
```

Comment:
This approach is readable and less verbose, but its lack of typing doesn't prevent omissions/typos like a nested/typed map such as the following would:
```
{
  [A in VEHICLE_EVENT]: {
    [B in VEHICLE_EVENT]: boolean
  }
}
```

Reply:
I was also wondering about the type safety. Strictly speaking this is basically a copy of the existing data structure, so I wonder if maybe it should use `Partial`, but then again it's generated by creating the full set of combos between states/transitions so this particular instance would be fully defined.